_storage/overview.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# A decentralized storage architecture
+
+The Blockstack Network stores application data using a storage service called
+Gaia. Transactional metadata is stored on the Blockstack blockchain and user
+application data is stored in Gaia hub. Storing data off of the blockchain
+ensures that Blockstack applications can provide users with high-performance and
+high availability for data reads and writes without introducing central trust
+parties.
+
+* TOC
+{:toc}
+
+##  Understand Gaia in the Blockstack architecture
+
+The following diagram depicts the Blockstack architecture and Gaia's place in it:
+
+{% include architecture.md %}
+
+## User control or how is Gaia decentralized?
+
+Within the provider's storage system a Gaia hub runs as a service. This service
+authenticates writes to a location by requiring a valid authentication token from a requestor.
+
+![Gai storage](/storage/images/gaia-storage.png)
+
+Gaia's approach to decentralization focuses on user control of data and storage.
+If a user can choose which storage provider to store data with, then that is all the decentralization required to enable user-controlled
+applications.
+
+The control of user data lies in the way that user data is accessed.
+When an application fetches a file `data.txt` for a given user `alice.id`, the
+lookup will follow these steps:
+
+1. Fetch the `zonefile` for `alice.id`
```

Comment:
Should we mention "...via BNS"?

Reply:
Not at this point; it would shift the focus.